### DISCUSSION TITLE
Getting an error when trying to create a window on Linux

### DISCUSSION THREAD

**Discussion by frazchaudhry** [dotnet/Silk.NET#477] [Q&A] [answered]
Hi,
    I'm trying out Silk.NET for the first time and going through the examples. I'm on Pop OS 20.10. I have dotnet sdks 3.1.408 and 5.0.202 installed. When trying to run the first example I get the following error.

Unhandled exception. System.NotSupportedException: The currently bound window platform(s) only support views, instead of windows. Use the view APIs instead.
   at Silk.NET.Windowing.Window.Create(WindowOptions options)
   at OpenGL.SilkNet.Program.Main(String[] args) in /home/fraz/Programs/Tutorials/CSharp/OpenGL.SilkNet/OpenGL.SilkNet/Program.cs:line 17

Process finished with exit code 134.

I switched to Windows 10 and am able to get a window to show without any problems. Is there anything I'm missing on LInux? Any dependencies?

Thank you

**Comment by Perksey** [comment, marked as answer]
Try installing libglfw3 using `sudo apt-get install libglfw3`

**Reply by frazchaudhry** [reply]
awesome. that did it. Thank you
